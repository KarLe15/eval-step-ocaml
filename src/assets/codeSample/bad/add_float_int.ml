let _ =
  40. + 2
