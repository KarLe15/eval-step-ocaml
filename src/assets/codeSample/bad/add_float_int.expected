File "", line 2, characters 2-5:
Error: This expression has type float but an expression was expected of type
         int
